src/components/donation-targets/index.tsx

```diff
@@ -1,17 +1,26 @@
+import Icon from 'components/icon'
 import Progress from 'components/progress'
-import Charity from 'orm/charity'
+import Charity, { DonationTarget } from 'orm/charity'
 import { createElement, FunctionComponent } from 'react'
+import { cadToHashes, formatCurrency, formatNumber } from 'utils'
 import styles from './style.scss'
 
 interface SectionProps {
-    title: string
-    value: number
-    max: number
+    target: DonationTarget
 }
 
-const Section: FunctionComponent<SectionProps> = props => (
+const Section: FunctionComponent<SectionProps> = ({ target }) => (
```

Comment:
Explanation of why this matters to put the event target here

Reply:
This is the donation target not an event target